.travis.yml

```diff
@@ -1,10 +1,12 @@
 dist: bionic
+
 addons:
   chrome: stable
-language: ruby
 
-notifications:
-  email: false
+services:
```

Comment:
This may not be needed here. If the current builds pass, I'll try yanking it out and re-pushing.